lib/elixir/lib/task.ex

```diff
@@ -434,11 +434,8 @@ defmodule Task do
   @doc false
   # TODO: Remove on 2.0
   # (hard-deprecated in elixir_dispatch)
-  def find(tasks, msg) do
-    do_find(tasks, msg)
-  end
-
-  defp do_find(tasks, {ref, reply}) when is_reference(ref) do
+  def find(tasks, msg)
```

Comment:
I added later thinking `h` command in IEx could show some information, but it doesn't. So I guess you are correct.